core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -402,8 +402,8 @@ refresh_account_db(Database) ->
     AccountId = kz_util:format_account_id(Database, 'raw'),
     _ = remove_depreciated_account_views(AccountDb),
     _ = ensure_account_definition(AccountDb, AccountId),
-    Views = get_all_account_views(),
-    _ = kapps_util:update_views(AccountDb, Views, 'true'),
+    _ = kapps_util:update_views(AccountDb, get_all_account_views(), 'true'),
+    _ = knm_maintenance:update_number_services_view(AccountDb),
```

Comment:
@fenollp i would prefer to split knm_maintenance:update_number_services_view if needed to have a fun that returns the view json, so it can be called from `kapps_maintenance:fetch_all_account_views`